src/Exception.cpp

```diff
@@ -1,5 +1,9 @@
 // Copyright 2019 André Hodapp
 #include "yacx/Exception.hpp"
+#if _MSC_VER
+#define popen _popen
+#define pclose _pclose
+#endif
```

Comment:
I didn't put this in it's own file, but i think it's fine here?

Reply:
Kein Plan, was bringt denn dieses popen und pclose?

Reply:
eh glaube das ist so um Prozesse zu starten und hier speziell um zu gucken wie breit die Shell ist oder so.
das geht nicht weil tput kein Windows Befehl ist, aber ich glaube ich benutze das auch in den Tests um zu gucken wie die GPU heißt und das habe ich gefixt. naja glaube das ist okay so